### DISCUSSION TITLE
Employee management: Take over vacation days

### DISCUSSION THREAD

**Discussion by HeinzWuert** [kimai/kimai#4542] [Plugins]
Is there a way to take over vacation days into the new year in a similar way to taking over overtime?

**Comment by kevinpapst** [comment]
https://demo.kimai.org/de/contract
<img width="1139" alt="Bildschirmfoto 2024-01-08 um 20 48 48" src="https://github.com/kimai/kimai/assets/533162/c8991b00-f8f3-458c-a536-a024825b2732">

**Comment by HeinzWuert** [comment]
Thanks!
